cmd/k8s-bigip-ctlr/main.go

```diff
@@ -26,12 +27,15 @@ import (
 	"time"
 
 	"github.com/F5Networks/k8s-bigip-ctlr/pkg/appmanager"
+	"github.com/F5Networks/k8s-bigip-ctlr/pkg/health"
 	"github.com/F5Networks/k8s-bigip-ctlr/pkg/pollers"
+	bigIPPrometheus "github.com/F5Networks/k8s-bigip-ctlr/pkg/prometheus"
 	"github.com/F5Networks/k8s-bigip-ctlr/pkg/vxlan"
 	"github.com/F5Networks/k8s-bigip-ctlr/pkg/writer"
 
 	log "github.com/F5Networks/k8s-bigip-ctlr/pkg/vlogger"
 	clog "github.com/F5Networks/k8s-bigip-ctlr/pkg/vlogger/console"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
```

Comment:
If not already done, we need to get legal to sign off on any additional 3rd party packages.

Reply:
I'll take care of that. I'd like to get this merged, but it's a WIP and I don't know why.

Reply:
@f5gary I resolved the WIP status (sorry that I forgot the change that).

Reply:
Thank you @johscheuer.

Reply:
FYI, I have verified we do not have any new 3rd party packages.